src/wasm-lib/src/lib.rs

```diff
@@ -196,3 +195,110 @@ pub async fn lsp_run(config: ServerConfig) -> Result<(), JsValue> {
 
     Ok(())
 }
+
+#[cfg(target_arch = "wasm32")]
+#[wasm_bindgen]
+pub fn is_points_ccw(points: &[f64]) -> i32 {
+    utils::is_points_ccw_wasm(points)
+}
+
+#[derive(Copy, Clone)]
+#[cfg(target_arch = "wasm32")]
+#[wasm_bindgen]
+pub struct Xy {
```

Comment:
Can we call this Point2d or sometjing we have something like this elsewhere if you want to reuse?

Reply:
Thanks.
I only created this for the wasm bridge, I ended up just flattening it all in 6c8ae2f9933103bef78edb6ab51d76abb89d8e4a and removed this struct altogether.
These language bridges are always a bit messy.